From 6c174fbbf4be7fc79eda0e9c1e6b86831ffea3f0 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Fri, 7 May 2021 17:46:01 -0400
Subject: [PATCH 296/296] Linux 5.10.41-rt42 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9..8bdfb9a 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt42
-- 
2.7.4

