From 55b00ef5e489f18183ab389de5d24ded4a6efae2 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 13:59:59 -0700
Subject: [PATCH 100/830] _____ANNOTATION: iommu changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index 63468c0..96ed870 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -3,3 +3,4 @@ Android vendor hooks
 kbuild changes
 x86 changes
 arm and arm64 changes
+iommu changes
-- 
2.7.4

