From acc12a18eeb71ceaafb8b90edfa28c324bec4f78 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 14:00:54 -0700
Subject: [PATCH 316/830] _____ANNOTATION: tty changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index b09c23c..17c5329 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -14,3 +14,4 @@ security changes
 binder changes
 ion changes
 driver framework and misc driver changes
+tty changes
-- 
2.7.4

